Sources/ChatUI/ChatInChannel/MessageList.swift

```diff
@@ -25,7 +25,7 @@ public struct MessageList<MessageType: MessageProtocol & Identifiable, RowConten
 
     @Environment(\.appearance) var appearance
     
-    @State private var isKeyboardShown = false
+    @State private var isKeyboardShown = true
```

Comment:
Please let me know the details on the issue regarding `isKeyboardShown`.👀
When I checked some chat services such as Instagram, WhatsApp, the default state of the keyboard is hidden.
If you need to change the default value of it, what about adding the parameter (e.g., `showsKeyboard` to the initializer of the message list.
Also, could you request these changes with a new separated PR? Because theses changes are not related to the purpose of this PR.
Thanks for the contributions⭐️